### ISSUE TITLE
[BUG] x-logging.FireLens.Advanced.FirelensConfiguration.Options.enable-ecs-log-metadata: false does not work

### ISSUE THREAD

**Issue by thorfi** [compose-x/ecs_composex#709]
**Describe the bug**
[BUG] x-logging.FireLens.Advanced.FirelensConfiguration.Options.enable-ecs-log-metadata: false does not work

**To Reproduce**
1. pip install
2. executed as CLI tool
3. Docker compose file
```
services:
  foo:
    x-logging: # ecscomposex
      RetentionInDays: 1
      FireLens:
        Advanced:
          FirelensConfiguration:
            Options:
              enable-ecs-log-metadata: false
          ComposeXManagedAwsDestinations:
            - log_group_name: "foo/"
```
4. Generated service.yml file has incorrect  enable-ecs-log-metadata value
```
Resources:
  ...
  EcsTaskDefinition:
    Properties:
      ContainerDefinitions:
        - ...
        - Command:
            Ref: AWS::NoValue
          Cpu: 102
          DependsOn:
            - Condition: SUCCESS
              ContainerName: log_router_preload
          DockerLabels:
            container_name: log_router
          ...
          FirelensConfiguration:
            Options:
              config-file-type: file
              config-file-value: /compose_x_rendered/firelens.conf
              enable-ecs-log-metadata: true
            Type: fluentbit
```

**Expected behavior**

service.yml enable-ecs-log-metadata setting should match the docker-compose.yml setting

**Logs**
Not Applicable.

**Desktop**
 - OS: macOS 14.1 (Sonoma)
 - Python version (3.11)
 - Fork https://github.com/thorfi/ecs_composex.git
 - Commit ID: 7ef7967aeb43609014bcab4b8ca6beb8dbc966a5

**Comment by JohnPreston** [comment]
Thanks @thorfi 
You're on fire!  This one sounds like an oversight

**Comment by JohnPreston** [comment]
EDIT: joking aside, I had so much forgotten about this parameter, I was recently looking into how to disable it :trollface:

**Comment by JohnPreston** [comment]
Okay, I have it working using your compose example
```yaml
          FirelensConfiguration:
            Options:
              config-file-type: file
              config-file-value: /compose_x_rendered/firelens.conf
              enable-ecs-log-metadata: true
            Type: fluentbit
          HealthCheck:
            Command:
              - CMD-SHELL
              - 'echo ''{"health": "check"}'' | nc 127.0.0.1 8877 || exit 1'
            Interval: 10
            Retries: 3
            StartPeriod: 15
            Timeout: 5
          Image:
            Ref: FluentBitAwsImage

```

**Comment by JohnPreston** [comment]
@thorfi will wait for your feedback if you don't mind testing out from #711

**Comment by JohnPreston** [comment]
FYI @thorfi I published 0.23.27rc4
Although I think that 0.23.27 will be out today

**Comment by thorfi** [comment]
Hey, I can confirm this is doing the right thing in the CF template. Thanks! :-)
